### DISCUSSION TITLE
How to use hibernate sequence in Planetscale

### DISCUSSION THREAD

**Discussion by two10** [planetscale/discussion#295] [Q&A] [answered]
While create a deploy request to main branch it says provide non null and primary key in every table , but in one of the table created by hibernate is hibernate_sequence which just have one coulmn , in vitess doc it says create use vschema 

but in dicussion it says planetscale dont currently support sharding right now , what to do

**Comment by fooforge** [comment, marked as answer]
Hi @two10,

I believe, me and you are also talking about this issue in a ticket you have created via [our Support Portal](https://support.planetscale.com/)? If so and if this is about the [Hibernate ORM](https://hibernate.org/orm/), I think, it's relatively safe to say that we don't seem to be compatible with Hibernate yet, I'm afraid.

If you could share any code or constructs generated by Hibernate with us through the ticket so that we can look into this further, that would be great. Thanks!
